### ISSUE TITLE
[question] Does rubrical detect the base language version?

### ISSUE THREAD

**Issue by nicoarg** [ivanklee86/rubrical#248]

**Comment by ivanklee86** [comment]
Rubrical does not detect base language (today).  Since it acts primarily on dependency definition files (e.g. pyproject.toml, go.mod, package.json), the language version would need to be part of the specification (e.g. for Poetry and `pyproject.toml`).

Do you have some examples where you’d like to check versions?

**Comment by nicoarg** [comment]
Yes, I do. I just have a bunch of reports written in different languages (Python, Go, TypeScript, C++, Java, and more!) and I need to determine which language and version they are using. With Github's API this is easy but to determine the version will depend on exploring different files (like `pyproject.toml` if the repository uses Python)
